develop/codingguidelines/Protocol/Primary_keys1.md

```diff
@@ -11,3 +11,5 @@ uid: Primary_keys1
 - Numeric keys are more efficient than long non-numeric keys. In order to improve performance, the use of an auto increment key should be considered instead of long non-numeric keys, if applicable.
 
 - Backslashes are not allowed in the primary keys because MySQL does not allow these.
+
+- Leading and trailing spaces should be avoided in primary keys (as well as in display keys)
```

Comment:
Leading and **_trailing_** spaces should be avoided _**in**_ primary keys (as well as **_in_** display keys)